test/node/test_farm.rb

```diff
@@ -107,4 +116,26 @@ def test_drops_expired_scores_from_history
       end
     end
   end
+
+  def test_garbage_farm_file
+    log = SaveLastMessageLogger.new
+    Dir.mktmpdir 'test' do |dir|
+      file = File.join(dir, 'corrupted_farm')
+      [
+        "0/6: 2018-06-26ABCT00:32:43Z 178.128.165.12 4096 MIRhypo1@c13620484b46caa4\n",
+        "some garbage\n"
+      ].each do |score_garbage_line|
+        File.write(file, score_garbage_line)
+        score = Zold::Score.new(
+          Time.parse('2017-07-19T21:24:51Z'),
+          'some-host', 9999, 'NOPREFIX@ffffffffffffffff', %w[13f7f01 b2b32b 4ade7e],
+          strength: 6
+        )
+        File.write(file, score_garbage_line + score.to_s)
```

Comment:
@emilianodellacasa it looks like you're writing `score_garbage_line` twice? Is it because score is necessary?

Reply:
No, I am writing a valid score and a garbage score, to check that just the invalid one is rejected

Reply:
@emilianodellacasa doesn't it mean that `score_garbage_line` is a valid line actually?

Reply:
No, _score_garbage_line_ is invalid (and will be excluded from score list), whereas the other one is valid and will stay in the list.

Reply:
@emilianodellacasa you're writ ing `score_garbage_line` twice, where the other one is coming from?

Reply:
@emilianodellacasa please use `puts` to write both lines, so that it's clear that those are two separate lines.